helm-chart/renku-core/templates/deployment.yaml

```
@@ -33,7 +33,7 @@ spec:
         app.kubernetes.io/name: {{ include "renku-core.name" $ }}
         app.kubernetes.io/instance: {{ $.Release.Name }}
         app.kubernetes.io/deploymentVersion: {{ $version.name }}
-        {{ include "call-nested" (list $ "redis" "redis.fullname") }}-client: "true"
+        {{ .Values.global.redis.clientLabel | default (printf "%s-%s" (include "redis.host" .) "-client") }}: "true"
```

```suggestion
        {{ $.Values.global.redis.clientLabel | default (printf "%s-%s" (include "redis.host" $) "-client") }}: "true"
```

Everything needs a `$` here. I missed that the whole deployment is in one `range` statement.